src/pages/ErrataFeed.vue

```diff
@@ -65,6 +65,21 @@
         binary-state-sort
         wrap-cells
       >
+        <template v-slot:top-right v-if="userAuthenticated()">
+          <q-btn
+            size="80%"
+            no-caps
+            icon="restart_alt"
+            color="grey-8"
+            @click="showDialogAdvisories = true"
+          >
+            Reset
+            <q-tooltip
+        <q-tooltip>
```

Comment:
Please remove one of the tooltips do the correct indent and make sure that everything works fine